dotCMS/src/main/java/com/dotmarketing/filters/CMSFilter.java

```diff
@@ -54,8 +54,22 @@ public void init(FilterConfig arg0) throws ServletException {
 
     }
 
+    
     public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
             throws IOException, ServletException {
+        
+        try {
+            _doFilter(req,res,chain);
+        }
+        finally {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this "finally" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)